joy_lin/alpha_challenges/conditionals.md

```diff
@@ -27,13 +43,27 @@ Create a Ruby method called `temperature_status` that takes three arguments: the
   - If the air conditioner is non-functional and the current temperature is below the the desired temperature, print the string "Fix the A/C whenever you have the chance. It's cool."
 
 ```ruby
-# write your code here
+def temperature_status(curr_temp, is_functional, desired_temp)
+  if (is_functional == true) && (curr_temp > desired_temp)
```

Comment:
Since `is_functional` is a boolean, you don't need to evaluate it with a comparator; you can just do:
```
if is_functional && (curr_temp > desired_temp)
```
If you're checking for false status, you can throw a bang in front of it (i.e. `!true` evaluates to `false`).

Reply:
got it!